packages/studio-ui/src/web/views/FlowBuilder/nodeProps/ActionModalForm.tsx

```diff
@@ -99,13 +99,18 @@ class ActionModalForm extends Component<Props, State> {
 
   prepareActions() {
     this.setState({
-      avActions: (this.props.actions || []).map(x => {
-        return {
-          label: x.name,
-          value: x.name,
-          metadata: { ...x }
-        }
-      })
+      avActions: (this.props.actions || [])
+        // hide module actions on cloud-enabled bots
+        .filter(action =>
+          !this.props.isCloudBot ? true : action.name.indexOf('builtin') === 0 || action.name.indexOf('/') === -1
```

Comment:
No need for ternary operator:
```ts
!this.props.isCloudBot || action.name.indexOf('builtin') === 0 || action.name.indexOf('/') === -1
```

Also, **personal pref**: I would extract 2 small private functions `isBuiltinAction()` and `isModuleAction()` so your condtion looks like this:
```ts
!this.props.isCloudBot || isBuiltinAction(action) || !isModuleAction(action)
```